client/lib/wp/sync-handler/whitelist-handler.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Module dependencies
+ */
+
+import debugFactory from 'debug';
+const debug = debugFactory( 'calypso:sync-handler:whitelist' );
+
+const whitelist = [
+	'^/batch',
+	'^/me',
+	'^/products$',
+	'^/read',
+	'^/sites/.+/'
```

Comment:
Well, let's just add the ones we're ready to test...  I think we should test the post-list endpoints like I mentioned above. We can add any others as long as they are fully tested.